images/dind/Dockerfile

```diff
@@ -72,6 +78,8 @@ RUN systemctl enable docker.service
 RUN echo "DOCKER_STORAGE_OPTIONS=--storage-driver vfs" >\
  /etc/sysconfig/docker-storage
 
+COPY openshift-dind-lib.sh /usr/local/bin/
```

Comment:
The dependency graph is base -> node -> master, so the master gets anything on the node image.